src/components/calculator.js

```diff
@@ -1,30 +1,52 @@
 import React from 'react'; // eslint-disable-line
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  async displayBoard(e) {
+    await this.setState(calculate(this.state, e.target.innerHTML));
+    if (this.state.total !== null && this.state.next === null) {
+      document.querySelector('.display-board').innerHTML = this.state.total;
+    } else if (this.state.next !== null && this.state.total !== null) {
+      document.querySelector('.display-board').innerHTML = this.state.next;
+    } else if (this.state.next !== null && this.state.total === null) {
+      document.querySelector('.display-board').innerHTML = this.state.next;
+    } else if (this.state.next === null && this.state.total === null) {
+      document.querySelector('.display-board').innerHTML = '0';
+    }
```

Comment:
- [ ] Your application looks great already. When I clicked any operate button twice or more times, I received this error : `Unhandled Rejection (Error): [big.js] Invalid number` . Please, let's create an exception to handle this scenario.

Reply:
Thanks for finding that BUG 🐛 , now I create an exception in the code to fix it. 😄